sections/shared/components/GasPriceSelect/GasPriceSelect.tsx

```diff
@@ -36,22 +31,24 @@ const GasPriceSelect: FC<GasPriceSelectProps> = ({ gasPrices, transactionFee, ..
 	const [gasSpeed, setGasSpeed] = useRecoilState<keyof GasPrices>(gasSpeedState);
```

Comment:
Can delete?
```suggestion
```

Reply:
I removed the set method, which is not required.